srunner/scenarioconfigs/osc2_scenario_configuration.py

```diff
@@ -312,22 +303,22 @@ def visit_event_declaration(self, node: ast_node.EventDeclaration):
 
             return ("event", (event_name, arguments))
             """
-            
-        def visit_method_declaration(self,  node: ast_node.MethodDeclaration):
+
+        def visit_method_declaration(self, node: ast_node.MethodDeclaration):
             pass
 
         def visit_method_body(self, node: ast_node.MethodBody):
             type = node.type
             for child in node.get_children():
                 if isinstance(child, ast_node.BinaryExpression):
                     self.visit_binary_expression(child)
-        
+
         def visit_binary_expression(self, node: ast_node.BinaryExpression):
             arguments = [self.visit_children(node), node.operator]
             flat_arguments = flat_list(arguments)
             temp_stack = []
             for ex in flat_arguments:
-                if ex == '+' or ex == '-' or ex == '*' or ex == '/' or ex == '%' :
+                if ex == '+' or ex == '-' or ex == '*' or ex == '/' or ex == '%':
```

Comment:
how about following code:
`if ex in ['+', '-', '*', '/', '%']`

Reply:
> how about following code: `if ex in ['+', '-', '*', '/', '%']`
OK, it's an optimal solution.